acct-user/boinc/boinc-0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for sci-misc/boinc"
+ACCT_USER_ID=405
+ACCT_USER_HOME=/var/lib/${PN}
+ACCT_USER_HOME_PERMS=0750
+ACCT_USER_GROUPS=( boinc video )
+
+acct-user_add_deps
+
+DEPEND+=" acct-group/boinc "
+RDEPEND+=" acct-group/boinc "
```

Comment:
This is not needed, because `acct-user_add_deps` adds `acct-group/*` dependencies deduced from `ACCT_USER_GROUPS`.

Reply:
Oh, that is good to know.
Actually I just copied that from an ebuild in the tree. :shrug: ( acct-user/kismet )

Reply:
Done